### PR TITLE
Fix error queue

### PR DESCRIPTION
**Pull request by manilde**
error queue: pod stopt met de melding 'connection already closed'  bij een lang durende manage.py commando.

### CODE REVIEW

File: src/statistiek_hub/resources/observation_resource.py [hunk 1]
```diff
@@ -62,16 +62,10 @@ def clean(self, value, row, **kwargs):
         start_date = convert_to_date(row["temporal_date"])
         end_date = add_timedelta(start_date, temporal_type)
 
-        temporal, created = TemporalDimension.objects.get_or_create(
+        temporal = TemporalDimension.objects.get(
```
Comment (vanbuiten) [inline src/statistiek_hub/resources/observation_resource.py:65]: The code now retrieves an existing TemporalDimension from the database, which means we lose the ability to create a new one if it doesn't exist. If this is intentional, what should happen when an item is not found?  Perhaps add a try-except block to log this error when it occurs?

**Review by vanbuiten — CHANGES_REQUESTED** [review]
Could you add a test for the custom DatabaseWrapper?

Also added some comments to the PR. Let me know when I can take a second look.

File: src/statistiek_hub/resources/observation_resource.py (hunk 1 above)
Comment (manilde) [reply src/statistiek_hub/resources/observation_resource.py:65]: the check if a TemporalDimension item exists is already done earlier and raises an exception.

File: src/custom_db_backend/base.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import django.db
+from django.contrib.gis.db.backends.postgis.base import (
+    DatabaseWrapper as BuiltinPostgresDatabaseWrapper,
+)
+from django.utils.asyncio import async_unsafe
+from psycopg2 import InterfaceError
+
+
+@async_unsafe
```
Comment (vanbuiten) [inline src/custom_db_backend/base.py:14]: This should be the decorator on the `create_cursor` class method

**Pull request merged**